### PR TITLE
add HTTP 3 425 Too Early status

### PR DESCRIPTION
**Pull request by kkmuffme**
Trac: https://core.trac.wordpress.org/ticket/60942

### CODE REVIEW

File: src/wp-includes/functions.php [hunk 1]
```diff
@@ -1408,6 +1408,7 @@ function get_status_header_desc( $code ) {
 			422 => 'Unprocessable Entity',
 			423 => 'Locked',
 			424 => 'Failed Dependency',
+			425 => 'Too Early',
```
Comment (mukeshpanchal27) [inline src/wp-includes/functions.php:1412]: Add `@since` annotation for new header type. Also needs to fix the unit test.

Comment (kkmuffme) [reply src/wp-includes/functions.php:1412]: Thanks, added. 

Which unit test? I checked a couple but they all fail due to unrelated reasons and seem to be broken in general not just for this PR.

**Comment by joemcgill** [comment]
Thanks @kkmuffme. Is there a related Trac ticket that this should be attached to? If not, would you mind opening one?

**Comment by kkmuffme** [comment]
Created and assigned the ticket https://core.trac.wordpress.org/ticket/60942

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r57936](https://core.trac.wordpress.org/changeset/57936).

**Pull request closed without merging**